zulipterminal/config/keys.py

```diff
@@ -244,6 +248,11 @@ class KeyBinding(TypedDict):
         'excluded_from_random_tips': True,
         'key_category': 'searching',
     },
+    'CLEAR_SEARCH': {
+        'keys': ['esc'],
+        'help_text': 'Clear search in current panel',
+        'key_category': 'searching',
+    },
```

Comment:
Since we're aiming at contextual help messages, should we be using the word `panel`?  It feels more like code terminology and might not be useful as a help message.

Reply:
Hmm.
Found mentions of 'panel' in the user facing documentation.
What other word do you think would fit better in this context?

Reply:
I found mentions of 'panel' in the user-facing documentation,
and as we discussed, it's hard to come up with suitable alternatives,
so I've left it as 'panel'.